README.md

```diff
@@ -15,7 +15,7 @@ Zivid Python is the official Python package for Zivid 3D cameras. Read more abou
 
 ### Dependencies
 
-* [Python](https://www.python.org/) version 3.6 or higher
+* [Python](https://www.python.org/) version 3.6, 3.7, 3.8, 3.9 and 3.10
```

Comment:
Slight complication to consider:
- We don't support 3.6 on Windows anymore (in the sense that we don't build, test & upload a .whl file for that).  This is because 3.6 has been end-of-life for a while now.
- However, we do test Python 3.6 on Linux, because that's the default Python on Ubuntu 18.04 and we still support that.

Reply:
Also we support `3.11` on `master` now. Just hasn't been released to PyPI yet. So 3.11 should be listed here.

PyPI displays a snapshot of the Readme on the commit that is released.

Reply:
Opinions, @nedrebo ?
Should this list show the versions that we support somewhere, or just the versions that we support everywhere?

Reply:
Let's just do the union. Should now be "3.6, 3.7, 3.8, 3.9, 3.10 and 3.11"

Reply:
I think we should just drop 3.6 as it is already been EOL for over a year. Even 3.7 is almost EOL too

Reply:
It's the default Python on Ubuntu 18.04, so I suppose we should drop it when U18 is dropped.

Reply:
Let's do the ones that are fully supported. I.e that `pip install` will work for on the listed versions.

Drop 3.6 here, but keep testing on 3.6 to keep undocumented support best effort until we hit a real blocker for that support or until Zivid SDK drops support for 18.04.